### ISSUE TITLE
Python Environment Setup

### ISSUE THREAD

**Issue by drgardner11** [huseinzol05/Stock-Prediction-Models#33]
Hi, 

I am running your jupyter notebook, the models I have tested, are all working, however I am getting slightly different results to you. Given that the models and data are the same I am assuming that there is something different with my Python Environment. In a separate issue, I have found the below package versions listed:
seaborn==0.9.0
pandas==0.23.4
numpy==1.14.5
matplotlib==3.0.2

Are you able to let me know the other package versions you are using please? Keras, Tensorflow, python and anything else that you think is significant. 

Thanks

**Comment by drgardner11** [comment]
Following on from the above, I have noticed that when I run the models multiple times, I get different results on each run. This could explain why I can't match the original results. The below outputs are from 1.lstm. Results 5 & 7 in particular are very different from all of the others. Any suggestions please?
Thanks

1:
![image](https://user-images.githubusercontent.com/49395983/59617264-ee52e580-911d-11e9-9ce8-319ff01356e2.png)

2:
![image](https://user-images.githubusercontent.com/49395983/59617362-1e9a8400-911e-11e9-8b89-86c413745c57.png)

3:
![image](https://user-images.githubusercontent.com/49395983/59617375-22c6a180-911e-11e9-9e8c-b4a90abacd47.png)

4:
![image](https://user-images.githubusercontent.com/49395983/59617394-2b1edc80-911e-11e9-86b9-52132afb3cb9.png)

5:
![image](https://user-images.githubusercontent.com/49395983/59617406-3245ea80-911e-11e9-80ef-1706d61dc38f.png)

6:
![image](https://user-images.githubusercontent.com/49395983/59617418-3a9e2580-911e-11e9-9db8-2a364953d8b5.png)

7:
![image](https://user-images.githubusercontent.com/49395983/59617431-412c9d00-911e-11e9-9c97-e9fa61c46517.png)

8:
![image](https://user-images.githubusercontent.com/49395983/59617455-4984d800-911e-11e9-9f85-c0a177399de2.png)

**Comment by huseinzol05** [comment]
It is totally depend on random seeds and entropy. It is normal not same. Some initiates might be bias, some might be sucks.

**Comment by huseinzol05** [comment]
improve deep learning models and added TF seed.
